python/lsst/afw/geom/point.cc

```diff
@@ -48,7 +48,6 @@ py::class_<PointBase<T,N>> declarePointBase(py::module &mod, const std::string &
     cls.def("le", [](PointBase<T,N> &p, T value) { return p.le(value); });
     cls.def("gt", [](PointBase<T,N> &p, T value) { return p.gt(value); });
     cls.def("ge", [](PointBase<T,N> &p, T value) { return p.ge(value); });
-    cls.def("__str__", &PointBase<T,N>::toString);
```

Comment:
SWIG had a very different wrapper for `__str__` and `__repr__` and my changes restore the old ones. Not only were the old ones nice, but at least one was required for a unit test to pass.

In my opinion our pybind11 wrappers are somewhat lacking in this area; I have seen several instances where we used to have nice readable representations that are now gone. I'll try to remember to document those and file a ticket. That said, the most worrisome changes are when the default representation is used, which is essentially useless. This one is more subtle since it was simply a change from one reasonable representation to another.